lib/main/Middlewares/ST/STM32_USB_Device_Library/Class/CDC_HID/Src/usbd_cdc_hid.c

```diff
@@ -0,0 +1,332 @@
+/*
```

Comment:
Same applies for this file - it doesn't belog here

Reply:
OK. I use the development as base.
I don't know why it is there.